cli/azd/internal/repository/detect_confirm.go

```diff
@@ -175,6 +183,8 @@ func (d *detectConfirm) render(ctx context.Context) error {
 			recommendedServices = append(recommendedServices, "Azure Database for PostgreSQL flexible server")
 		case appdetect.DbMongo:
 			recommendedServices = append(recommendedServices, "Azure CosmosDB API for MongoDB")
+		case appdetect.DbRedis:
```

Comment:
Can we have tests for app-detect, calling `DetectProject()` from all the supported projects to add coverage and protect againts regressions?

Reply:
We do have these tests, I can update

Reply:
done